multichain-testing/package.json

```
@@ -0,0 +1,92 @@
+{
+  "name": "@agoric/multichain-testing",
+  "version": "0.1.0",
+  "description": "E2E Interchain testing environment for Agoric smart contracts.",
+  "type": "module",
+  "scripts": {
+    "build": "exit 0",
+    "lint": "yarn lint:eslint",
+    "lint:eslint": "eslint .",
+    "lint:types": "tsc --noEmit",
+    "lint-fix": "yarn lint:eslint --fix",
+    "test": "ava",
+    "postinstall": "patch-package",
```

Yarn 4 has its own patching protocol
```suggestion
    "postinstall": "patch-package",
```
Also remove the dep